modules/dlscript/api_generator.cpp

```diff
@@ -92,6 +93,23 @@ List<ClassAPI> generate_c_api_classes() {
 	List<StringName> classes;
 	ClassDB::get_class_list(&classes);
 
+	// Register global constants as a fake GlobalConstants singleton class
+	{
+		ClassAPI global_constants_api;
+		global_constants_api.class_name = L"GlobalConstants";
+		global_constants_api.api_type = ClassDB::API_CORE;
```

Comment:
Not sure the value this field should have...
May it's better to keep it at `API_NONE` to give a way to determine it's not a real class.